tests/polkadotjs_test/test/test-polkadot.js

```diff
@@ -180,7 +179,22 @@ describe('Testing polkadot.js/api calls:', function () {
             expect(transfer).to.have.lengthOf(32);
         });
     });
-    describe('api grandpa', () => {
+    describe('api state', () => {
+        it('call api.rpc.state.queryStorage()', async function () {
+            const block0Hash = await api.rpc.chain.getBlockHash(0);
+
+            const value = await
+                api.rpc.state.queryStorage(["0x1cb6f36e027abb2091cfb5110ab5087f06155b3cd9a8c9e5e9a23fd5dc13a5ed",
+                    "0xc2261276cc9d1f8598ea4b6a74b15c2f57c875e4cff74148e4628f264b974c80"], block0Hash);
+            expect(value).to.be.not.null;
+            value.forEach(item => {
+                expect(item[0]).to.have.lengthOf(32);
+                expect(item[1]).to.have.lengthOf(2);
```

Comment:
Can these be predicted or not? 🤔

Reply:
no, the value seems to be random.

Reply:
Do you think you could regex check them? Just to be sure we don't return i.e. empty strings (especially for future code changes).